packages/lib-user/src/components/Contributors/components/ContributorsList/ContributorsList.js

```diff
@@ -0,0 +1,93 @@
+import { Box } from 'grommet'
+import { arrayOf, number, shape, string } from 'prop-types'
+
+import MemberStats from '../MemberStats'
+import ProjectStats from '../ProjectStats'
+
+function ContributorsList({
+  contributors = [],
+  projects = []
+}) {
+  return (
+    <Box
+      as='ol'
+    >
+      {contributors.map((contributor, index) => {
+        const totalHoursSpent = contributor.session_time >= 0 ? contributor.session_time / 3600 : 0
```

Comment:
Is there a similar helper calculation for showing a signed-in user their hours of participation at `/users/[login]/stats`? I assume there's a similar calculation for building a printable certificate too. I suggest you create a shared helper function or file for "hours" calculations to be sure the relationship between number of classifications, session time, and the display hours in the UI is consistent.

Also, in this component specifically when would `contributor.session_time` not be `>= 0`? Is it undefined in some cases?

Reply:
1. Yes, the related user stats calculation is in MainContent - https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-user/src/components/shared/MainContent/MainContent.js#L43

2. A shared helper is a good idea, I'll include in the certificate page work

3. When developing the ProfileHeader I added the ERAS time check for greater than 0, but it might not be necessary. I created a new user and the user stats response for the user stats page included time values as `0` and a new user group was similar. I can revisit and refactor with helper function with certificate page work if that's ok